compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingAttrs.cpp

```diff
@@ -99,6 +311,33 @@ EncodingAttr::verify(function_ref<mlir::InFlightDiagnostic()> emitError,
       }
     }
   }
+  if (iterationSizesAttr) {
+    if (!userIndexingMapsAttr) {
+      return emitError()
+             << "found `iteration_sizes` without no `user_indexing_maps`";
```

Comment:
question: I'm not a native speaker, but should it be `with no` or `without any`?

Reply:
Yes, indeed, went for `without any`.